src/workloads/query/ExpressiveQueries.yml

```diff
@@ -19,15 +19,13 @@ Actors:
 
 - Name: InsertData
   Type: Loader
-  # Loader doesn't support multiple threads for a single collection.
-  # TODO: TIG-2938 Use multiple threads to speedup insertion.
-  Threads: 1
+  Threads: 4
   Phases:
   - Repeat: 1
     Database: &DB test
-    Threads: 1
+    MultipleThreadsPerCollection: true
     CollectionCount: 1  # Collection name will be Collection0, this is not configurable.
-    DocumentCount: 1e6
```

Comment:
[optional nit] I would revert this change as it's easier to read '6' than to count six zeros.

Reply:
I agree, but for when MultipleThreadsPerCollection is true, yaml parsing is forcing this number to be an integer because it is doing integer division of DocumentCount / ThreadCount. 

We can address it in C++ code, but to keep the PR smaller, I decided to just change it to integer.

Reply:
The fix for it is actually pretty easy, but I would like to do it in a separate PR

Reply:
Sounds good; thanks for addressing this!